### PR TITLE
PR for #32: Adjust text size of the test results

### LINKED ISSUE #32: Option for adjusting text size of the test results in the plot
Create an option for adjusting the text size of the test results in the plot.

### PR DESCRIPTION
**Pull request by rayhuang11**
Closes #32

### CODE REVIEW

**Review by Constantino-Carreto-Romero — COMMENTED** [review]
Great job, it looks good. It only remains to add a description in the help file of xteventplot.

**Comment by rayhuang11** [comment]
@Constantino-Carreto-Romero thanks! I've edited the file in https://github.com/JMSLab/xtevent/pull/48/commits/87a96a4bc910c6a64c42abb210c1c88cac120c0f. If it looks good to you, we can go ahead and merge this branch.

**Comment by Constantino-Carreto-Romero** [comment]
Excellent @rayhuang11  it looks good. Just let's wait for what @jorpppp thinks of the implementation in the meeting, before the merge. 
About the description in the help file, it might also be a good idea a description of the values the option fontzise can receive, or a link to the size function.

**Comment by rayhuang11** [comment]
@Constantino-Carreto-Romero thanks! Good idea. I've added the link to the size function.

**Comment by jorpppp** [comment]
@rayhuang11  's solution works well to adjust the size of the test results in the plots, but there is no reason to limit ourselves to just modifying the size. We are going to generalize this approach to allow for any text formatting option in these test results.

File: xtevent/xteventplot.sthlp [hunk 1]
```diff
@@ -205,6 +205,10 @@ overlay plot. These options are only active if {opt overlay(trend)} is specified
 {phang2}{cmd:. xteventplot, smpath(line)}{p_end}
 {phang2}{cmd:. xteventplot, smpath(line, technique(nr 10 bfgs 10))}{p_end}
 
+{pstd}Adjust textbox options for the p-values of the pre-trend and leveling-off tests{p_end}
+{phang2}{cmd:. xteventplot, textboxoption(size(tiny))}{p_end}
+{phang2}{cmd:. xteventplot, textboxoption(color(blue) size(large))}{p_end}
```
Comment (jorpppp) [inline xtevent/xteventplot.sthlp:210]: We only need one example.

**Comment by jorpppp** [comment]
We only need one adjustment to the examples and then this PR is ready to merge.

**Comment by rayhuang11** [comment]
@jorpppp ready to merge!

**Pull request merged**